.github/workflows/swift-test-linux.yml

```
@@ -23,13 +23,20 @@ jobs:
           # Use dbus-run-session to wrap the test execution
           dbus-run-session -- sh -c '
             echo "test" | gnome-keyring-daemon --unlock --components=secrets
             ./Scripts/tsan-test.sh
           '
+      - name: Test VeinSCUI
+        run: |
+          dbus-run-session -- sh -c '
+            echo "test" | gnome-keyring-daemon --unlock --components=secrets
+            TEST_SCUI=1 ./Scripts/tsan-test.sh
+          '
+TEST_SCUI=1 ./Scripts/tsan-test.sh
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)